ParisBNF/abb/BNFabb97.xml

```diff
@@ -0,0 +1,1357 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb97" xml:lang="en" type="mss">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+    <fileDesc>
+      <titleStmt>
+        <title>Glory of the Kings, Documents on the History of ʾAksum and beyond</title>
+        <editor role="cataloguer" key="NV"/>
+      </titleStmt>
+      <publicationStmt>
+        <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+        <pubPlace>Hamburg</pubPlace>
+        <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+          Forschungsumgebung / Beta maṣāḥǝft</publisher>
+        <availability>
+          <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+            <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0. </p>
+          </licence>
+        </availability>
+
+      </publicationStmt>
+      <sourceDesc>
+        <msDesc xml:id="ms">
+          <msIdentifier>
+            <repository ref="INS0303BNF"/>
+            <collection>Manuscrits orientaux</collection>
+            <collection>Fonds éthiopien</collection>
+            <collection>d'Abbadie</collection>
+            <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100875850">BnF Éthiopien d'Abbadie
+              97</idno>
+          </msIdentifier>
+          <msContents>
+            <msItem xml:id="ms_i1">
+              <locus from="2r" to="81rb"/>
+              <title type="complete" ref="LIT1709Kebran"/>
+              <textLang mainLang="gez"/>
+              <note>The manuscript was not used in <bibl>
+                  <ptr target="bm:Bezold1905KebraN"/>
+                </bibl>. The titles of the 118 chapters are included in the text.</note>
+              <msItem xml:id="ms_i1.1">
+                <title type="complete" ref="LIT1709Kebran#Chapter1"/>
+                <textLang mainLang="gez"/>
```

Comment:
`  <textLang mainLang="gez"/>` in children msItems after a parent msItem that already contains it is not necessary (it's not wrong, just for the future if you would like the file to be less bulky)

Reply:
> ` <textLang mainLang="gez"/>` in children msItems after a parent msItem that already contains it is not necessary (it's not wrong, just for the future if you would like the file to be less bulky)

deleted!